go.mod

```diff
@@ -46,7 +46,7 @@ require (
 	github.com/aliyun/alibaba-cloud-sdk-go v1.61.1059
 	github.com/apparentlymart/go-cidr v1.1.0
 	github.com/aws/amazon-ec2-instance-selector/v2 v2.0.2
-	github.com/aws/aws-sdk-go v1.40.38
+	github.com/aws/aws-sdk-go v1.42.5
```

Comment:
Maybe it needs a rebase?
https://github.com/kubernetes/kops/commit/0c08f047ba38639837a7b40e62a01b3cd51edf6c

Reply:
Probably not for the last time before this can go in...

We could merge this one and always set DNS64 to false though.

Reply:
Probably a good idea.